docker-compose-base.yml

```diff
@@ -4,6 +4,7 @@ services:
     command: /app-dev/dockerDev.sh
     volumes:
     - ./packages:/packages
+    - /mnt/filestore:/filestore
```

Comment:
so I see /mnt/filestore is being used here but /filestore is used [in docker-compose.yml](https://github.com/UCSC-MedBook/MedBook_/blob/searchable-jobs/docker-compose.yml)

Can you confirm my understanding is correct :
/filestore is mounted as a drive on prod and staging hosts, but not on dev hosts. and docker-compose-base only gets used when running from devStart. So this is a compromise to allow a filestore on dev hosts without having to have the /filestore drive
?